pkg/apis/extensions/v1alpha1/types_operatingsystemconfig.go

```diff
@@ -146,6 +146,9 @@ type File struct {
 	Permissions *uint32 `json:"permissions,omitempty"`
 	// Content describe the file's content.
 	Content FileContent `json:"content"`
+	// NodeName contains the name of the node for node-specific configurations.
+	// If NodeName is not empty the corresponding file will only be rolled out to the node with the specified name.
+	NodeName string `json:"nodeName,omitempty"`
```

Comment:
Should this have a pointer and `optional` tag?
```suggestion
	// +optional
	NodeName *string `json:"nodeName,omitempty"`
```